src/node/desktop/src/main/gwt-callback.ts

```diff
@@ -57,6 +57,14 @@ export class GwtCallback extends EventEmitter {
     super();
     this.owners.add(mainWindow);
 
+    const fontDescriptors = [
+      ...new Map<string, IQueryFontDescriptor>(findFontsSync({}).map((fd) => [fd.family, fd])).values(),
+    ].sort((a, b) => a.family?.localeCompare(b.family ?? '') ?? 0);
+    const monospaceFonts = fontDescriptors.filter((fd) => fd.monospace).map((font) => font.family);
+    const proportionalFonts = fontDescriptors.filter((fd) => !fd.monospace).map((font) => font.family);
+
+    // this.monospaceFonts = [...new Set(fontDescriptors)].sort((a, b) => a.family.localeCompare(b.family));
```

Comment:
Either remove this commented out line or add a comment why you want to keep it around.

Reply:
Sorry, I missed this one when cleaning everything up